scripts/build_llvm.sh

```diff
@@ -182,3 +191,12 @@ else
   cp bin/llvm-lit "$LLVM_INSTALL_PREFIX/bin/"
   cd "$working_dir" && echo "Installed llvm build in directory: $LLVM_INSTALL_PREFIX"
 fi
+
+if $compiler_rt; then
+  cd $llvm_source/build && ninja runtimes && ninja install-runtimes
+  status=$?
+  if [ "$status" = "" ] || [ ! "$status" -eq "0" ]; then
+    echo "Build failed. Please check the files in the `pwd`/logs directory."
+    cd "$working_dir" && if $is_sourced; then return 1; else exit 1; fi
+  fi
```

Comment:
```suggestion
  else
    cd "$working_dir" && echo "Successfully added runtime components."
  fi
```
Please cd back to the working dir - nothing more annoying that a script that dumps me in a "random" directory once it is done. ;)